tools/tracegen/CFileWriter.cpp

```diff
@@ -45,7 +45,9 @@ CFileWriter::writeOutputFiles(J9TDFOptions *options, J9TDFFile *tdf, J9TDFGroup
 	time_t targetFileMtime = FileUtils::getMtime(fileName);
 
 	if ((false == options->force) && (targetFileMtime > sourceFileMtime)) {
-		printf("C file is already up-to-date: %s\n", fileName);
+		if (true == options->verboseOutput) {
```

Comment:
since the variable is a boolean can you remove the `true == ` part?

Reply:
Fixed. There are also some cases I saw where `false == options->force` could be replaced with `!options->force` if we wanted to.